### ISSUE TITLE
Not able to run PointScalar with query variables

### ISSUE THREAD

**Issue by fmaillet24** [EverWinter23/graphene-gis#9]
Hello @EverWinter23 , 

I'm trying to work with query variables and point scalar for many hours, and always got the same issue... Do you have any clue to give me ?

thanks a lot

<img width="1440" alt="Screenshot 2021-05-22 at 6 15 34 PM" src="https://user-images.githubusercontent.com/32547247/119233538-f695d580-bb29-11eb-8290-f5fa6171ccc6.png">

**Comment by EverWinter23** [comment]
Hmm, could you share the `Type` or `schema.py`?

Do checkout this example project-> https://github.com/EverWinter23/geoql, More specifically [this](https://github.com/EverWinter23/geoql/blob/master/formula1/mutation.py) file. Maybe it'll help?

**Comment by Ouradze** [comment]
I think I have found the bug. I will try to tests it this week.

**Comment by EverWinter23** [comment]
Lemme know if you still face issues.

**Comment by EverWinter23** [comment]
Closing, need more info.

**Comment by Dimitri-Gnidash** [comment]
This is still an issue. It has to do with the expectation that the value of point scalar is expected to be quoted but there is no way to pass that in JSON.

**Comment by Dimitri-Gnidash** [comment]
The issue happens in `GISScalar` class in `parse_value` method.
When the value is passed in via variable, the `parse_value` method is called instead of `parse_literal`. However, the node type at that point is `str` not `ast.StringValue`. Strings do not have `value` attribute and that's the error.
```
    @classmethod
    def parse_value(cls, node):
        geometry = GEOSGeometry(node.value)
        return json.loads(geometry.geojson)
```
https://github.com/EverWinter23/graphene-gis/blob/52fd4ad83935333b434c83258ab00a89aae8e79d/graphene_gis/scalars.py#L26

The solution is to change parse_value to:
```
    @classmethod
    def parse_value(cls, node: str):
        geometry = GEOSGeometry(node)
        return json.loads(geometry.geojson)
```

**Comment by EverWinter23** [comment]
@Dimitri-Gnidash Thanks for the info! I'll try to verify the issue and patch it.

**Comment by EverWinter23** [comment]
This has been fixed and released in version `0.0.7`. On another note, maybe the following might help someone:

```python
"""
NOTE: Refer https://github.com/EverWinter23/graphene-gis/issues/9
Some evasive issue. I wouldn't recommend using the following for
implementing mutations, because it involves some unnecessary steps.

The converter will convert the WKT geometry to geojson, which you'll
have to convert back to WKT using GEOSGeometry to initialize the object.

It's way more simpler to use the graphene.String as the input argument for 
WKT inputs. This is demonstrated in the next test case.
"""
def test_scalar_query_variables():
    class Query(graphene.ObjectType):
        point_model = graphene.Field(PointModelType, location=PointScalar(required=True))

        def resolve_point_model(self, info, location):
            return PointModel(location=GEOSGeometry(f"{location}"), props={"type": "Feature"})

    schema = graphene.Schema(query=Query)

    query = """
        query TestQuery($location: PointScalar!) {
            pointModel(location: $location) {
                location
                props
            }
        }
    """

    expected = {
        "pointModel": {
            "location": {
                "type": "Point",
                "coordinates": [42.0, 15.0]
            },
            "props": {
                "type": "Feature"
            }
        }
    }

    result = schema.execute(query, variables={"location": "POINT(42.0 15.0)"})
    assert not result.errors
    assert json.loads(json.dumps(dict(result.data))) == expected


def test_native_query_variables():
    class Query(graphene.ObjectType):
        point_model = graphene.Field(PointModelType, location=graphene.String(required=True))

        def resolve_point_model(self, info, location):
            return PointModel(location=location, props={"type": "Feature"})

    schema = graphene.Schema(query=Query)

    query = """
        query TestQuery($location: String!) {
            pointModel(location: $location) {
                location
                props
            }
        }
    """

    expected = {
        "pointModel": {
            "location": {
                "type": "Point",
                "coordinates": [42.0, 15.0]
            },
            "props": {
                "type": "Feature"
            }
        }
    }

    result = schema.execute(query, variables={"location": "POINT(42.0 15.0)"})
    assert not result.errors
    assert json.loads(json.dumps(dict(result.data))) == expected
```
